tonytanaka/index.js

```diff
@@ -0,0 +1,17 @@
+// Q01
+console.log("There were sold", 6450 * 15, "tickets");
```

Comment:
Here we need subtraction and NOT multiplication.
console.log("There were sold", 6450 / 15, "tickets");